lib/folio_sync/archives_space_to_folio/folio_synchronizer.rb

```diff
@@ -29,14 +41,23 @@ def sync_resources_to_folio
         Dir.foreach(marc_dir) do |file|
           next if ['.', '..'].include?(file)
 
-          Rails.logger.debug "Processing file: #{file}"
-          bibid = File.basename(file, '.xml')
+          bib_id = File.basename(file, '.xml')
```

Comment:
If the ASpace record is new, the bibid won't be available yet. I think there are a few different ways of handling this when it comes to actually naming files, which would impact how errors are logged, so I'm going to hold off on offering a suggestion!

Reply:
I think that in a later version of the code, once we add the create-FOLIO-record-if-not-exist logic, we'll have the bibid (a.k.a. FOLIO instance record hrid) before we download the ASpace MARC, so this should be okay.  There's a chance we might want to use the term "hrid" instead of "bibid" at that point though.

Reply:
Good point - currently we don't have logic for creating new records. For this version of the script, I raise an error if the `bib_id` is nil when saving a MARC record from ASpace (and this error will be caught and included in the error report). 
Also, I just moved the line above (extracting basename) into the begin/rescue statement to ensure that all errors are caught.